plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1713,7 +1723,7 @@ public HiveInsertTableHandle beginInsert(ConnectorSession session, ConnectorTabl
                         columnTypes,
                         getColumnStatistics(partitionComputedStatistics, ImmutableList.of()));
 
-                if (partitionUpdate.getUpdateMode() == OVERWRITE) {
+                if (partitionUpdate.getUpdateMode() == PartitionUpdate.UpdateMode.OVERWRITE) {
```

Comment:
I see there are two  OVERWRITE variables now, but maybe add an import for `PartitionUpdate.UpdateMode`?